server/routes/routers/follows.ts

```diff
@@ -130,4 +130,34 @@ follow.get('/isFollowing/:followingId', async (req: any, res: any) => {
     await prisma.$disconnect();
   }
 });
+
+follow.get('/counts/:id', async (req, res) => {
+  const { id } = req.params;
+
+  try {
+    const userData = await prisma.user.findUnique({
+      where: { id: +id },
+      include: {
+        followedBy: {
+          select: { id: true },
+        },
+        following: {
+          select: { id: true },
+        },
+      },
+    });
+
+    if (userData) {
+      const followersCount = userData.followedBy.length;
+      const followingCount = userData.following.length;
```

Comment:
You can use `_count` in your prisma query to count the number of followers and following. Docs [here](https://www.prisma.io/docs/orm/prisma-client/queries/aggregation-grouping-summarizing#count-relations)